data/sgb-words.txt

```diff
@@ -5754,4 +5754,4 @@ osier
 roble
 rumba
 biffy
-pupal
```

Comment:
Got rid of the newline at the end of this file because it added an extra line to the scores file.. there are other ways I could get around the issue though; some folks prefer newlines at the end of all files 🤷🏻